<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="QWizard" name="Wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff;</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NASA-TLX</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>857</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Bitte wählen Sie jeweils die Dimension aus, die für die Durchführung der Aufgabe wichtiger war.</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>631</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="12" column="1">
      <widget class="QSlider" name="mental_physical">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Frustration</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>Geistige Anforderung</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Leistung</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Zeitliche Anforderung</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Frustration</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Frustration</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSlider" name="performance_time">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>Leistung</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>geistige Anstrengung</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Anstrengung</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="performance_frustration">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Körperliche Anforderung</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QSlider" name="effort_physical">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>Geistige Anforderung</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSlider" name="performance_mental">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSlider" name="frustration_effort">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="time_effort">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSlider" name="physical_frustration">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Körperliche Anforderung</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="time_frustration">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSlider" name="mental_effort">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Anstrengung</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QSlider" name="frustration_mental">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Anstrengung</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="effort_performance">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Körperliche Anforderung</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSlider" name="time_mental">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Zeitliche Anforderung</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Leistung</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSlider" name="physical_time">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Frustration</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Körperliche Anstrengung</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Anstrengung</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Körperliche Anstrengung</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Leistung</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="physical_performance">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Zeitliche Anforderung</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Frustration</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Leistung</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Geistige Anforderung</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Zeitliche Anforderung</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Zeitliche Anforderung</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Anstrengung</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Geistige Anforderung</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
